webapp/sources/api-doc/paths/campaigns/saveEvent.yml

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+post:
+  summary: Get a campaign event details
+  description: Get a campaign event details
```

Comment:
I'm not sure about that one. Does it save the future event based on an existing campaign ?